pkg/views/workspace/create/build_choice.go

```diff
@@ -5,32 +5,32 @@ package create
 
 import "fmt"
 
-type BuildChoice string
+type BuilderChoice string
```

Comment:
I suggest you revert this back to `BuildChoice` since the prop in the project that this controls is called `Build`

Reply:
@Tpuljak This file is more so linked to "Cobra" than to the "views" folder as it's used in the command help/validation and not in views.
However, I believe moving it might create some import cycle issue because of the enum values. Should we move it to a new location outside of `views` or keep it as is? The `cmd/workspace/util` scope comes to mind

Reply:
I'm fine with it staying here because it's related to this view. I would just rename it back to `BuildChoice`